src/services/Utils.js

```diff
@@ -417,7 +417,6 @@ class Utils {
         const savedFilter = getSavedFilter();
         query = {
           ...savedFilter,
-          page: 1,
```

Comment:
Seems like it was not just for fun here? Are you sure that this is not used anywhere else?